src/client.rs

```diff
@@ -612,16 +612,16 @@ impl RequestBuilder {
     ///
     /// let client = reqwest::Client::new()?;
     /// let res = client.post("http://httpbin.org")
-    ///     .json(&map)
+    ///     .json(&map)?
     ///     .send()?;
     /// # Ok(())
     /// # }
     /// ```
```

Comment:
I think I can reuse [the explanation](https://docs.serde.rs/serde_json/fn.to_vec.html#errors) that is in `serde_json` for this.